### DISCUSSION TITLE
Une seule VM avec docker-compose ?

### DISCUSSION THREAD

**Discussion by TWR-RWT** [DstMlOpsCrypto/MainCrypto#39] [General]
Suite au meeting du 16.08.2024 avec Sebastien, je me dis qu'on devrait répondre à cette question: est ce que l'on commence d'abord par faire notre sur système à partir d'une seule VM avec un docker-compose, avant d'expérimenter l'architecture à 3 VM ?

**Comment by TWR-RWT** [comment]
Par ailleurs j'ai déjà mis un docker-compose à la racine pour l'API. Il suffit d'y rajouter vos services.

**Comment by FredericLAGNIEZ** [comment]
Une première version du projet à une VM me semble pertinent, ainsi on pourra tous tester le projet de manière asynchrone
